themes/pom.xml

```diff
@@ -73,6 +78,17 @@
                 <directory>src/main/resources-community</directory>
             </resource>
         </resources>
+        <plugins>
+            <plugin>
+                <artifactId>maven-source-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>**/node_modules/**/*</exclude>
+                        <exclude>**/web_modules/**/*</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+        </plugins>
```

Comment:
Does this end up excluding node_modules/web_modules from the JAR?

Reply:
Yes, it excludes the modules. I didn't consider them sources, but dependencies. It make the artifact smaller, and it speeds up the build.

Reply:
Adding to this: They are removed from the source JAR, which should IMHO contain only real sources, not dependencies or generated files.